analysis/src/CompletionFrontEnd.ml

```diff
@@ -743,6 +743,12 @@ let completionWithParser1 ~currentFile ~debug ~offset ~path ~posCursor
       mbs |> List.iter scopeModuleBinding;
       mbs |> List.iter (fun b -> iterator.module_binding iterator b);
       processed := true
+    | Pstr_include {pincl_mod = {pmod_desc = med}} -> (
+      match med with
+      | Pmod_apply ({pmod_desc = Pmod_ident {txt = lid; loc}}, _) ->
```

Comment:
Looks like this only covers the functor form `include M(...)` but not simple module include `include M`.